src/components/CommunityStatus.tsx

```diff
@@ -0,0 +1,225 @@
+import i18n from 'assets/i18n';
+import BigNumber from 'bignumber.js';
+import { amountToCurrency } from 'helpers/currency';
+import {
+    calculateCommunityProgress,
+    calculateCommunityRemainedFunds,
+} from 'helpers/index';
+import { CommunityAttributes } from 'helpers/types/models';
+import { IRootState } from 'helpers/types/state';
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import { Title, ProgressBar, Text, Divider } from 'react-native-paper';
+import { useSelector } from 'react-redux';
+import { ipctColors } from 'styles/index';
+
+import SuspiciousCard from './SuspiciousCard';
+import Card from './core/Card';
+import WarningRedTriangle from './svg/WarningRedTriangle';
+
+interface ICommuntyStatusProps {
+    children?: any; // linter issues are a bit anoying sometimes
+    community: CommunityAttributes;
+}
+
+export default function CommunityStatus(props: ICommuntyStatusProps) {
+    const { community } = props;
+    const { width } = Dimensions.get('screen');
+    const [communityGoal, setCommunityGoal] = useState<BigNumber>();
+    const [remainedFunds, setRemainedFunds] = useState<number>();
+    const user = useSelector((state: IRootState) => state.user.metadata);
+
+    const app = useSelector((state: IRootState) => state.app);
+
+    useEffect(() => {
+        const goal = new BigNumber(community.contract.maxClaim).multipliedBy(
+            community.state.beneficiaries
+        );
+        const days = calculateCommunityRemainedFunds(community);
+        setCommunityGoal(goal);
+        setRemainedFunds(days);
+    }, [community]);
+
+    if (community.contract === undefined || community.state === undefined) {
+        return null;
+    }
+
+    return (
+        <Card elevation={0} style={{ marginTop: 16 }}>
+            {community.suspect !== undefined && community.suspect !== null && (
+                <SuspiciousCard suspectCounts={community.suspect.suspect} />
+            )}
+            <View style={{ padding: 22 }}>
+                <View style={styles.cardWrap}>
+                    <View
+                        style={{
+                            flex: 1,
+                            alignItems: 'flex-start',
+                            justifyContent: 'center',
+                        }}
+                    >
+                        <Text
+                            style={[
+                                styles.description,
+                                {
+                                    color: ipctColors.regentGray,
+                                    fontSize: width < 375 ? 11 : 14,
```

Comment:
Not really, since we're using a 20% ratio on very small devices and it's has been very confusing to reach the same UI using size-matters for both lineHeight and fontSizes.